### PR TITLE
docs: redirect http -> https

### PR DESCRIPTION
**Pull request by mistercrunch**
### SUMMARY

because  Screen Shot 2020-09-15 at 5 53 13 PM does NOT look good!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
-<!--
-Licensed to the Apache Software Foundation (ASF) under one
-or more contributor license agreements.  See the NOTICE file
-distributed with this work for additional information
-regarding copyright ownership.  The ASF licenses this file
-to you under the Apache License, Version 2.0 (the
-"License"); you may not use this file except in compliance
-with the License.  You may obtain a copy of the License at
+[//]: # Licensed to the Apache Software Foundation (ASF) under one
```
Comment (mistercrunch) [inline docs/README.md:1]: gatsby-mdx was tripping on the comment...

File: docs/.htaccess [hunk 2]
```diff
@@ -0,0 +1,3 @@
+RewriteEngine On
```
Comment (mistercrunch) [inline docs/.htaccess:1]: here's the redirect magic

File: docs/README.md (hunk 1 above)
Comment (rusackas) [reply docs/README.md:1]: Hah, I have [another PR](https://github.com/apache/incubator-superset/pull/10898) open for some redirect support, and it seems like you can get away with just the `# ` for comments. Well, we'll see if it passes CI.

File: docs/README.md [hunk 3]
```diff
@@ -37,7 +36,7 @@ npm run start
 To publish, the static site that Gatsby generates needs to be pushed
 to the `asf-site` branch on the
 [apache/incubator-superset-site](https://github.com/apache/incubator-superset-site/)
-repository. No need to PR here, just `git push`!
+repository. No need to PR here, simply `git push`.
```
Comment (rusackas) [inline docs/README.md:39]: image

File: superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts [hunk 4]
```diff
@@ -64,7 +64,7 @@ describe('AdhocFilters', () => {
     });
   });
 
-  it('Set custom adhoc filter', () => {
+  xit('Set custom adhoc filter', () => {
```
Comment (rusackas) [inline superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts:67]: This PR just got a lot more controversial! I'm inclined to approve, but hope we can fix this test one day, as it HAS caught real errors for me before.

**Review by rusackas — APPROVED** [review]
The `xit` might be contentious, and I think you can simplify the apache boilerplate commenting, but in general the real fix here looks good, so, ✅

File: docs/README.md (hunk 1 above)
Comment (rusackas) [reply docs/README.md:1]: Hmmm. The comment format in this PR is still visible on the readme too... not sure what the trick is here.

image

File: superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts (hunk 4 above)
Comment (mistercrunch) [reply superset-frontend/cypress-base/cypress/integration/explore/AdhocFilters.test.ts:67]: I'M GOING CRAZY DEALING WITH FLAKY TESTS.

FYI that's the second flaky test in this module. Another one got fixed by @nytai yesterday

**Comment by mistercrunch** [comment]
I'd say merge this and let's get back to the xits

**Pull request merged**